src/task/CamelJBangTaskProvider.ts

```diff
@@ -71,4 +71,13 @@ export class CamelJBangTaskProvider implements TaskProvider {
 			return '';
 		}
 	}
+
+	private getRedHatMavenRepository(): string {
+		if (this.getCamelVersion().includes('redhat')) {
+			const url = workspace.getConfiguration().get('camel.debugAdapter.RedHatMavenRepository') as string;
+			return url ? `--repos=\$repos,${url}` : '';
```

Comment:
based on answer in https://issues.apache.org/jira/browse/CAMEL-19486 WDYT to conditionally set \$repos / #repos based on version provided :thinking:

Reply:
also here #repos used -> https://camel.apache.org/manual/camel-jbang.html#_placeholders_substitutes

Reply:
SKIP, I read more comments :rofl:

Reply:
the thing is "in newer versions" 😆 but thanks! you at least tried 👍🏻